README.md

```diff
@@ -231,25 +239,25 @@ git cf
-| `gcf`    | `FORGIT_CHECKOUT_FILE_GIT_OPTS`                                             |
-| `gcb`    | `FORGIT_CHECKOUT_BRANCH_GIT_OPTS`, `FORGIT_CHECKOUT_BRANCH_BRANCH_GIT_OPTS` |
-| `gbd`    | `FORGIT_BRANCH_DELETE_GIT_OPTS`                                             |
-| `gct`    | `FORGIT_CHECKOUT_TAG_GIT_OPTS`                                              |
-| `gco`    | `FORGIT_CHECKOUT_COMMIT_GIT_OPTS`                                           |
-| `grc`    | `FORGIT_REVERT_COMMIT_GIT_OPTS`                                             |
-| `gss`    | `FORGIT_STASH_SHOW_GIT_OPTS`                                                |
-| `gsp`    | `FORGIT_STASH_PUSH_GIT_OPTS`                                                |
-| `gclean` | `FORGIT_CLEAN_GIT_OPTS`                                                     |
-| `grb`    | `FORGIT_REBASE_GIT_OPTS`                                                    |
-| `gbl`    | `FORGIT_BLAME_GIT_OPTS`                                                     |
-| `gfu`    | `FORGIT_FIXUP_GIT_OPTS`                                                     |
-| `gcp`    | `FORGIT_CHERRY_PICK_GIT_OPTS`                                               |
+| Command  | Option                                                                       |
+|----------|------------------------------------------------------------------------------|
+| `ga`     | `FORGIT_ADD_GIT_OPTS`, `FORGIT_ADD_PREVIEW_GIT_OPTS`                         |
+| `glo`    | `FORGIT_LOG_GIT_OPTS`, `FORGIT_LOG_PREVIEW_GIT_OPTS`                         |
+| `gd`     | `FORGIT_DIFF_GIT_OPTS`                                                       |
+| `grh`    | `FORGIT_RESET_HEAD_GIT_OPTS`, `FORGIT_RESET_HEAD_PREVIEW_GIT_OPTS`           |
+| `gcf`    | `FORGIT_CHECKOUT_FILE_GIT_OPTS`, `FORGIT_CHECKOUT_FILE_PREVIEW_GIT_OPTS`     |
+| `gcb`    | `FORGIT_CHECKOUT_BRANCH_GIT_OPTS`, `FORGIT_CHECKOUT_BRANCH_BRANCH_GIT_OPTS`  |
+| `gbd`    | `FORGIT_BRANCH_DELETE_GIT_OPTS`                                              |
+| `gct`    | `FORGIT_CHECKOUT_TAG_GIT_OPTS`                                               |
+| `gco`    | `FORGIT_CHECKOUT_COMMIT_GIT_OPTS`, `FORGIT_CHECKOUT_COMMIT_PREVIEW_GIT_OPTS` |
+| `grc`    | `FORGIT_REVERT_COMMIT_GIT_OPTS`, `FORGIT_REVERT_PREVIEW_GIT_OPTS`            |
+| `gss`    | `FORGIT_STASH_SHOW_GIT_OPTS`, `FORGIT_STASH_SHOW_PREVIEW_GIT_OPTS`           |
+| `gsp`    | `FORGIT_STASH_PUSH_GIT_OPTS`, `FORGIT_STASH_PUSH_PREVIEW_GIT_OPTS`           |
+| `gclean` | `FORGIT_CLEAN_GIT_OPTS`, `FORGIT_CLEAN_PREVIEW_GIT_OPTS`                     |
+| `grb`    | `FORGIT_REBASE_GIT_OPTS`, `FORGIT_FILE_PREVIEW_GIT_OPTS`                     |
+| `gbl`    | `FORGIT_BLAME_GIT_OPTS`                                                      |
+| `gfu`    | `FORGIT_FIXUP_GIT_OPTS`, `FORGIT_FILE_PREVIEW_GIT_OPTS`                      |
+| `gcp`    | `FORGIT_CHERRY_PICK_GIT_OPTS`, `FORGIT_CHERRY_PICK_PREVIEW_GIT_OPTS`         |
```

Comment:
We should probably note down somewhere which git commands the `*_PREVIEW_GIT_OPTS` are used with to let people know which arguments they can use, e.g. it might not be obvious to everyone that `FORGIT_LOG_PREVIEW_GIT_OPTS` is used with `git show`. Might also make sense to break this out into a separate table if it does not fit into this one.